bucket/kaspersky-virus-removal-tool.json

```diff
@@ -0,0 +1,23 @@
+{
+    "version": "202311170938",
+    "description": "Kaspersky's Virus Removal Tool. Scan and disinfect an infected computer from viruses and other types of malicious programs.",
+    "homepage": "https://support.kaspersky.com/15674",
+    "license": "Freeware",
+    "url": "http://devbuilds.kaspersky-labs.com/devbuilds/KVRT/latest/full/KVRT.exe",
+    "hash": "8f7e31b2be8d7250d2920e50bf56098ed4d31832dd54058ca71f45cab640d7b9",
+    "bin": "KVRT.exe",
+    "shortcuts": [
+        [
+            "KVRT.exe",
+            "Kaspersky Virus Removal Tool",
+            "-accepteula"
+        ]
+    ],
+    "checkver": {
+        "url": "http://devbuilds.kaspersky-labs.com/devbuilds/KVRT/latest/full/kvrt.xml",
+        "regex": "\"([\\d]+)\""
+    },
+    "autoupdate": {
+        "url": "http://devbuilds.kaspersky-labs.com/devbuilds/KVRT/latest/full/KVRT.exe"
```

Comment:
Please change this to the correct https link:

https://devbuilds.s.kaspersky-labs.com/devbuilds/KVRT/latest/full/KVRT.exe